Draft: `act/honey` update with info about complaints (#844)
datenanfragen/website: content/en/act/honey/index.md

    "title": "Honey, the data hoarder—request a copy of your data now!",
    "type": "act",
    "date": "2020-11-02T14:43:34+02:00",

>     "last_edited": "2022-02-18T22:01:04+02:00",

Reviewer: ```suggestion
    "last_edited": "2022-02-26T19:55:04+02:00",
```

    "act_like": true
}

In our {{< link slug="blog/honey-data-collection" text="investigation on Honey" >}}, we have shown that the browser extension collects their users' data on a large scale. Regardless of whether an account has been registered or not, the add-on diligently logs page views on plenty of websites and sends them to the company behind the extension, {{< link slug="company/joinhoney" text="Honey Science LLC" >}}, a US company that was [bought by PayPal](https://help.joinhoney.com/article/302-what-does-honey-joining-paypal-mean-for-members).

{{< featuredImg alt="Photo of a lot of bees that rush to a drop of sugar water, above that the text: “Honey, the data hoarder—request your data now”" >}}

## What should I do?

Did you use Honey and maybe weren't aware of this data hoarding? Or do you simply want to know what Honey has saved about you? The GDPR grants you a number of {{< link slug="your-gdpr-rights" text="rights regarding your personal data" >}}, including the right to request a free copy of all data a company has collected <!-- stored? --> about you, the {{< link slug="your-gdpr-rights#right-of-data-access" text="**right of data access**" >}}.

> If you want to file a complaint with the data protection authorities, after inspecting the files Honey stored on you, you can use the template at the end of this article.

Reviewer: ```suggestion
If you feel Honey violated your rights after inspecting the data they have stored on you and you want to file a complaint with the data protection authorities, you can use the template at the end of this article.
```

    });
});
</script>

## How can I file a complaint about this?

> The GDPR gives you the right to file a complaint at a {{< link slug="supervisory-authorities" text="Supervisory authorities" >}} against a company.

Reviewer: ```suggestion
The GDPR gives you the right to file a complaint against a company with the {{< link slug="supervisory-authorities" text="supervisory authorities" >}}.
```

You can use this tool to find the authority responsible for you: 

<div class="sva-finder"></div>
<script>

>     window.props = {override: { country: { de: 'debralda' } }}

Reviewer: ```suggestion
    window.props = { override: { country: { de: 'debralda' } } };
```

    window.addEventListener('load', function() { renderSvaFinder(); });
</script>

> We also prepared a complaint template for you to use. Modify it accordingly to your situation.

Reviewer: ```suggestion
We also prepared a complaint template for you to use. Modify it according to your situation.
```

<p>If you need any more details, please feel free to contact me. You can reach me <span class="blog-letter-fill-in">enter your contact details here</span>.</p>

<p>Thanks in advance for your help.</p>

<p>Yours sincerely,<br>
<span class="blog-letter-fill-in">enter your name here</span></p>
</details>
</div>

You can use this tool to find the authority responsible for you and their contact details: 

<div class="sva-finder"></div>
{{< noScript "noscript-sva" >}}
<script>
    window.props = { override: { country: { de: 'debralda' } } };
    window.addEventListener('load', function() { renderSvaFinder(); });
</script>

The responsible authority will check your complaint and if they determine a misconduct on Honey's side, they can instruct them to cease that activity or even impose a fine.

Be prepared for this process to take a while. The authorities need will need time to process your complaint, especially if the company is in another country.

---

**Updates**:

We have made the following changes to this article since the initial publication:

> - [February 20, 2022](https://github.com/datenanfragen/website/pull/844): Restructured article and added a complaint template and reference to it.

Reviewer: ```suggestion
- [February 26, 2022](https://github.com/datenanfragen/website/pull/844): Restructured article and added a complaint template and reference to it.
```
